services/QuillLMS/spec/models/activity_spec.rb

```diff
@@ -96,6 +96,25 @@
     end
   end
 
+  describe '#fill_in_the_blank' do
+    it 'should default to false' do
+      new_activity = Activity.new
+      expect(new_activity.fill_in_the_blank).to be false
+    end
+
+    it 'should be settable to true' do
+      activity.fill_in_the_blank = true
+      activity.save!
+      expect(activity.reload.fill_in_the_blank).to be true
+    end
+
+    it 'should be settable to false' do
+      activity.fill_in_the_blank = false
+      activity.save!
+      expect(activity.reload.fill_in_the_blank).to be false
+    end
```

Comment:
I think that this test probably stands just fine as-is, but I do want to point out a weird thing going on here: if `fill_in_the_blank` is already `false`, this particular test doesn't actually confirm that this command changes things.  I think this is still probably good enough to cover most of what we want, but I wanted to flag just in case.